### PR TITLE
[extension] Fix issue if --yes is not provided while publish extension

### PR DESCRIPTION
**Pull request by Juliehzl**
When `if not yes` condition is not satisfied, the following error will occur:
```
local variable 'client' referenced before assignment
```

### CODE REVIEW

File: azdev/operations/extensions/__init__.py [hunk 1]
```diff
@@ -291,12 +291,12 @@ def publish_extensions(extensions, storage_account, storage_account_key, storage
     subheading('Uploading WHLs')
     for whl_path in whl_files:
         whl_file = os.path.split(whl_path)[-1]
+        from azure.storage.blob import BlockBlobService
+        client = BlockBlobService(account_name=storage_account, account_key=storage_account_key)
+        exists = client.exists(container_name=storage_container, blob_name=whl_file)
```
Comment (fengzhou-msft) [inline azdev/operations/extensions/__init__.py]: L296 can be moved into the not yes branch.

File: azdev/operations/extensions/__init__.py [hunk 2]
```diff
@@ -291,12 +291,12 @@ def publish_extensions(extensions, storage_account, storage_account_key, storage
     subheading('Uploading WHLs')
     for whl_path in whl_files:
         whl_file = os.path.split(whl_path)[-1]
+        from azure.storage.blob import BlockBlobService
```
Comment (haroldrandom) [inline azdev/operations/extensions/__init__.py]: Move import statement ourside the for loop

Comment (haroldrandom) [reply azdev/operations/extensions/__init__.py]: Fixed

File: azdev/operations/extensions/__init__.py (hunk 1 above)
Comment (haroldrandom) [reply azdev/operations/extensions/__init__.py]: If L296 and L297 moving into not yes branch, it means no change

Comment (Juliehzl) [reply azdev/operations/extensions/__init__.py]: If L296 moves to not yes branch, there is no client for exists().

**Pull request merged**